### PR TITLE
Iceberg data lake snowflake glue

### PR DESCRIPTION
**Pull request by sfc-gh-aengelbrecht**
Iceberg lab on how to use Snowflake with AWS and AWS Glue 

This is needed for the AWS Dev Days on April 10th - marketing requires the link by Friday April 5th

### CODE REVIEW

File: site/sfguides/src/data-lake-using-apache-iceberg-with-snowflake-and-aws-glue/data-lake-using-apache-iceberg-with-snowflake-and-aws-glue.md [hunk 1]
```diff
@@ -0,0 +1,533 @@
+author: Andries Engelbrecht
+id: data-lake-using-apache-iceberg-with-snowflake-and-aws-glue
```
Comment (iamontheinet) [inline site/sfguides/src/data-lake-using-apache-iceberg-with-snowflake-and-aws-glue/data-lake-using-apache-iceberg-with-snowflake-and-aws-glue.md]: Please replace "-" with "_" in the id "data-lake-using-apache-iceberg-with-snowflake-and-aws-glue"

File: site/sfguides/src/data-lake-using-apache-iceberg-with-snowflake-and-aws-glue/data-lake-using-apache-iceberg-with-snowflake-and-aws-glue.md [hunk 2]
```diff
@@ -0,0 +1,533 @@
...
+summary: This guide shows how to integrate Snowflake with External Volumes to S3 and the Glue Data Catalog for use with Iceberg tables
+<!--- Categories below should be hyphenated, i.e., Getting-Started. Do not leave blank. Visit site for available categories. -->
```
Comment (iamontheinet) [inline site/sfguides/src/data-lake-using-apache-iceberg-with-snowflake-and-aws-glue/data-lake-using-apache-iceberg-with-snowflake-and-aws-glue.md]: Please delete this comment.

File: site/sfguides/src/data-lake-using-apache-iceberg-with-snowflake-and-aws-glue/data-lake-using-apache-iceberg-with-snowflake-and-aws-glue.md [hunk 3]
```diff
@@ -0,0 +1,533 @@
...
+INSURER_NAME, INCEPTIONDATE, RENEWALDATE
+FROM QUOTE_ANALYSIS_ICE
+WHERE TOTALPREMIUMPAYABLE >100
+AND POLICY_STATUS_DESC = 'Renewed' 
+ORDER BY CREATEDDATE DESC;
+```
+
+
+<!-- ------------------------ -->
+## Conclusion
```
Comment (iamontheinet) [inline site/sfguides/src/data-lake-using-apache-iceberg-with-snowflake-and-aws-glue/data-lake-using-apache-iceberg-with-snowflake-and-aws-glue.md:518]: Please make this section similar to https://quickstarts.snowflake.com/guide/getting_started_with_dataengineering_ml_using_snowpark_python/index.html#8

**Review by iamontheinet — CHANGES_REQUESTED** [review]
@sfc-gh-aengelbrecht See review comments.

**Comment by sfc-gh-aengelbrecht** [comment]
Resolved comments, thanks!

**Pull request merged**